app/controllers/instance_actors_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class InstanceActorsController < ApplicationController
+  include AccountControllerConcern
+
+  before_action :set_cache_headers
+
+  def show
+    respond_to do |format|
+      format.html do
+        expires_in 0, public: true
+      end
+
+      format.json do
+        expires_in 3.minutes, public: true
+        render json: @account, content_type: 'application/activity+json', serializer: ActivityPub::ActorSerializer, adapter: ActivityPub::Adapter, fields: restrict_fields_to
+      end
+    end
+  end
+
+  private
+
+  def set_account
+    @account = Account.find(-99)
+  end
+
+  def restrict_fields_to
+    %i(id type preferred_username inbox public_key endpoints icon image url manually_approves_followers)
```

Comment:
What's the point of icon/image/url/manually_approves_followers?

Reply:
`icon` and `image` were set to banner and hero or something like that in the other PR, but I have not done that, so it's not super useful, I'll remove them.

`manually_approves_followers` is kinda here to not confuse people when follows do not work

`url` is used to point at an explanation of what the actor is.